identity/app/views/fragments/profile/digitalPackForm/details.scala.html

```diff
@@ -0,0 +1,105 @@
+<div class="is-hidden js-dig-info">
+    <ul class="details-list u-unstyled js-dig-details">
+        <li class="details-list__item">
```

Comment:
When you have a repeated pattern like this, you can reduce the noise by putting a "micro-template" inside the same template, e.g. https://github.com/guardian/membership-frontend/blob/master/frontend/app/views/patterns/patterns.scala.html#L13-L25